docs/testing-components/unit-testing/index.md

```diff
@@ -39,7 +39,7 @@ We use a dizzying number of tools to perform unit testing.  A quick overview of
 
 ## Unit Testing
 
-> If a video is more your style we have one here: [https://www.youtube.com/watch?v=DNAsXbnCV8w](https://www.youtube.com/watch?v=DNAsXbnCV8w)
+<!-- >> If a video is more your style we have one here: [https://www.youtube.com/watch?v=DNAsXbnCV8w](https://www.youtube.com/watch?v=DNAsXbnCV8w) -->
```

Comment:
Delete NOT comment out if this line is not needed.

Reply:
I deleted the comment.